pom.xml

```diff
@@ -154,13 +154,7 @@
             <dependency>
                 <groupId>com.metamx</groupId>
                 <artifactId>server-metrics</artifactId>
-                <version>0.2.8</version>
-                <exclusions>
-                    <exclusion>
-                        <groupId>com.metamx</groupId>
-                        <artifactId>java-util</artifactId>
```

Comment:
@drcrallen Druid doesn't depend on java-util directly. Most if not all deps which depend on it transitively use java-util:0.28.2, including server-metrics.